graphql/build.gradle

```diff
@@ -1,17 +1,21 @@
+plugins {
+    id "io.micronaut.build.internal.module"
+}
+
 dependencies {
-    annotationProcessor("io.micronaut:micronaut-inject-java:${micronautVersion}")
-    annotationProcessor("io.micronaut:micronaut-validation:${micronautVersion}")
+    annotationProcessor("io.micronaut:micronaut-inject-java")
```

Comment:
I think `annotationProcessor("io.micronaut:micronaut-inject-java")` is added automatically by the `io.micronaut.build.internal.module` plugin

Reply:
That's correct: https://github.com/micronaut-projects/micronaut-build/blob/278c63890a5457637a66eaef8d6dfe59addcf291/src/main/groovy/io/micronaut/build/MicronautModulePlugin.groovy#L22-L32

Reply:
👍 

I'm going to try to burn that into my memory 😀